src/main/flight/pid.c

```diff
@@ -723,6 +783,12 @@ static void pidRewrite(pidProfile_t *pidProfile, controlRateConfig_t *controlRat
         // -----calculate total PID output
         axisPID[axis] = PTerm + ITerm + DTerm;
```

Comment:
In comparison with other PIDs, D term seems to be inverted. Is this intentional?